<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaneCustomOrderParams</class>
 <widget class="QWidget" name="PaneCustomOrderParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fichiers commandes personnalisés</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonAdd">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRemove">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listViewCustoms">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pour faire une somme de colonne, séparer les noms de colonne par une virgule.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Le format de la date est celui de la documentation de la librairie C++ Qt.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTest">
       <property name="text">
        <string>TEST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableViewSelectedCustoms">
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
